src/test/rpc/ServerInfo_test.cpp

```diff
@@ -20,53 +20,31 @@
 #include <BeastConfig.h>
 #include <ripple/protocol/JsonFields.h>
 #include <test/jtx.h>
+#include <test/jtx/envconfig.h>
 #include <ripple/beast/unit_test.h>
-
-#include <boost/format.hpp>
+#include <ripple/core/ConfigSections.h>
 
 namespace ripple {
 
 namespace test {
 
-namespace validator {
+namespace validator_data {
 static auto const seed = "ss7t3J9dYentEFgKdPA3q6eyxtrLB";
 static auto const master_key =
     "nHUYwQk8AyQ8pW9p4SvrWC2hosvaoii9X54uGLDYGBtEFwWFHsJK";
 static auto const signing_key =
     "n9LHPLA36SBky1YjbaVEApQQ3s9XcpazCgfAG7jsqBb1ugDAosbm";
-// Format manifest string to test trim()
 static auto const manifest =
-    "    JAAAAAFxIe2cDLvm5IqpeGFlMTD98HCqv7+GE54anRD/zbvGNYtOsXMhAuUTyasIhvj2KPfN\n"
-    " \tRbmmIBnqNUzidgkKb244eP794ZpMdkYwRAIgNVq8SYP7js0C/GAGMKVYXiCGUTIL7OKPSBLS     \n"
-    "\t7LTyrL4CIE+s4Tsn/FrrYj0nMEV1Mvf7PMRYCxtEERD3PG/etTJ3cBJAbwWWofHqg9IACoYV\n"
-    "\t +n9ulZHSVRajo55EkZYw0XUXDw8zcI4gD58suOSLZTG/dXtZp17huIyHgxHbR2YeYjQpCw==\t  \t";
+    "JAAAAAFxIe2cDLvm5IqpeGFlMTD98HCqv7+GE54anRD/zbvGNYtOsXMhAuUTyasIhvj2KPfN"
+    "RbmmIBnqNUzidgkKb244eP794ZpMdkYwRAIgNVq8SYP7js0C/GAGMKVYXiCGUTIL7OKPSBLS"
+    "7LTyrL4CIE+s4Tsn/FrrYj0nMEV1Mvf7PMRYCxtEERD3PG/etTJ3cBJAbwWWofHqg9IACoYV"
+    "+n9ulZHSVRajo55EkZYw0XUXDw8zcI4gD58suOSLZTG/dXtZp17huIyHgxHbR2YeYjQpCw==";
```

Comment:
Why was this changed? It looks like the old test was testing for extra white space that this modified test no longer exercises.